docs/cloud/visualize/interact-new-charts.md

```diff
@@ -144,105 +148,154 @@ The following aggregate functions are available for each selected dimension:
 
 ### Reset to defaults
 
-Click on the 3-dot icon (**⋮**) on any chart, then **Reset to Defaults**, to reset the definition bar to its initial
-state.
+Finally, you can reset everything to its defaults by clicking the green "Reset" prompt at the end of the definition bar.
+
+## Anomaly Rate ribbon
+
+Netdata's unsupervised machine learning algorithm creates a unique model for each metric collected by your agents, using exclusively the metric's past data.
+It then uses these unique models during data collection to predict the value that should be collected and check if the collected value is within the range of acceptable values based on past patterns and behavior.
+
+If the value collected is an outlier, it is marked as anomalous.
+
+<img src="https://user-images.githubusercontent.com/70198089/236139886-79d63cf6-61ed-4aa7-842c-b5a1728c870d.png" width="900"/>
+
+This unmatched capability of real-time predictions as data is collected allows you to **detect anomalies for potentially millions of metrics across your entire infrastructure within a second of occurrence**.
+
+The Anomaly Rate ribbon on top of each chart visualizes the combined anomaly rate of all the underlying data, highlighting areas of interest that may not be easily visible to the naked eye.
+
+Hovering over the Anomaly Rate ribbon provides a histogram of the anomaly rates per presented dimension, for the specific point in time.  
+
+Anomaly Rate visualization does not make Netdata slower. Anomaly rate is saved in the the Netdata database, together with metric values, and due to the smart design of Netdata, it does not even incur a disk footprint penalty.
+
+## Hover over the chart
+
+Hovering over any point in the chart will reveal a more informative overlay.
+It includes a bar indicating the volume percentage of each time series compared to the total, the anomaly rate, and a notification on if there are data collection issues.
 
-## Jump to single-node dashboards
+This overlay sorts all dimensions by value, makes bold the closest dimension to the mouse and presents a histogram based on the values of the dimensions.
 
-Click on **X Charts**/**X Nodes** to display one of the two dropdowns that list the charts and nodes contributing to a
-given composite chart. For example, the nodes dropdown.
+<img src="https://user-images.githubusercontent.com/70198089/236141460-bfa66b99-d63c-4a2c-84b1-2509ed94857f.png" width="500"/>
 
-![The nodes dropdown in a composite chart](https://user-images.githubusercontent.com/1153921/99305049-7c019b80-2810-11eb-942a-8ebfcf236b7f.png)
+When hovering the anomaly ribbon, the overlay sorts all dimensions by anomaly rate, and presents a histogram of these anomaly rates.
 
-To jump to a single-node dashboard, click on the link icon 
-<img class="img__inline img__inline--link" src="https://user-images.githubusercontent.com/1153921/95762109-1d219300-0c62-11eb-8daa-9ba509a8e71c.png" /> next to the
-node you're interested in.
+#### Info column
```

Comment:
There's an Info bar at the bottom part of the chart as well that needs some explanation as you get different data when you hover over that.